relnotes/await-timeout.feature.md

```diff
@@ -0,0 +1,15 @@
+# Call `awaitOrTimeout` via scheduler interface
+
+Thanks to introducing `IScheduler` interface, it is not possible to call
+`awaitOrTimeout` as scheduler member method:
+
+```D
+import ocean.task.IScheduler;
+theScheduler.awaitOrTimeout(task, timeout);
+```
+
+Internally it still does the same call to global function in
+`ocean.task.util.Timer`, but new alias allows for slightly more uniform API.
+
+NB: this means that any app using the scheduler is now required to add
+`-L-lebtree` to its linker flags.
```

Comment:
Hm, so this is migration/feature note. Maybe add `migration` note with just this last paragraph as well.